nixos/modules/services/web-apps/filesender.nix

```diff
@@ -0,0 +1,261 @@
+{ config
+, lib
+, pkgs
+, ...
+}:
+let
+  format = pkgs.formats.php { finalVariable = "$config"; };
+
+  cfg = config.services.filesender;
+  simpleSamlCfg = config.services.simplesamlphp.filesender;
+  fpm = config.services.phpfpm.pools.filesender;
+
+  phpPackage = pkgs.php.withExtensions (
+    { enabled
+    , all
+    }: with all;
+    [
+      xml
+      mbstring
+    ]
+    ++ enabled
+  );
+
+  filesenderConfigPhpFile = format.generate "config.php" cfg.settings;
+
+  filesenderConfigDirectory = pkgs.stdenv.mkDerivation {
+    name = "filesender-config";
+    phases = [ "buildtree" ];
```

Comment:
Then we also don't need to overwrite phases here which should almost never be done.